README.md

```diff
@@ -186,6 +188,14 @@ Example:
 ["aar", { "reverseJetify": true }]
 ```
 
+#### `concurrent`
+
+Set to `true` to run all build stages concurrently.
+
+#### `failFast`
+
+If the `concurrent` option is `true`, setting this option to `true` will exit all builds immediately if any build fails. Ignored if `concurrent` is `false`.
+
```

Comment:
Is there any reason to provide options for these instead of making the builds work that way by default?